### DISCUSSION TITLE
How to integrate more than 1 10X multiome data for comparative analysis

### DISCUSSION THREAD

**Discussion by inayat45shaikh** [stuart-lab/signac#1516] [Q&A] [answered]
Hi,
We have done a multiomics profile (using 10X mutiome profiling kit) of two cell states in three cancer cell line systems and want to identify cellular population diversity (shared and unique cell types) between the cell states in each system. We have 2 replicates of each cell state in cancer system.
I have looked through different tutorials and all of them are for processing of single sample multiome data.
https://stuartlab.org/signac/articles/pbmc_multiomic
https://satijalab.org/seurat/articles/weighted_nearest_neighbor_analysis

I don't know what approach I should use for the question we are trying to answer (cell population diversity between the cell states in each system), as the above tutorials are for processing single sample and I probably would need to integrate these samples.
Also, i have 2 biological replicates for each cell state in each cell system, in this situation, what should be the idela way to process the data.

Thanks!
Inayat

**Comment by timoast** [comment, marked as answer]
If you have batch effects between the datasets that you want to remove, you can follow the integration tutorials showing various approaches to remove these. After that, you can follow a similar analysis workflow for analyzing one dataset.
